pkg/anonymization/anonymizer.go

```diff
@@ -128,6 +132,19 @@ func NewAnonymizerFromConfigClient(
 		networks = append(networks, networkRegex.FindAllString(installConfig, -1)...)
 	}
 
+	// egress subnets
+
+	hostSubnets, err := networkClient.HostSubnets().List(ctx, metav1.ListOptions{})
+	if err != nil {
+		return nil, err
+	}
+
+	for _, hostSubnet := range hostSubnets.Items {
```

Comment:
I'm pretty sure that the lint will complain about these two `for`, 'cause it will say that you should use `index` or `pointer`.

Reply:
Um, why?

Reply:
Check these links:
https://go-critic.github.io/overview#rangeValCopy-ref
https://stackoverflow.com/questions/20185511/range-references-instead-values